packages/components/src/FormField/FormField.tsx

```diff
@@ -200,7 +200,7 @@ export function FormField({
     shouldShowMiniLabel(defaultValue, value),
   );
   const [identifier] = useState(uuid.v1());
-  name = name || `generatedName--${identifier}`;
+  name = !name && validations ? `generatedName--${identifier}` : name;
```

Comment:
This ternary works but, it's a bit to parse how would you feel about an if statement instead? As `name` is already `name` if it's set.

Reply:
👍